torch/csrc/profiler/kineto_shim.cpp

```diff
@@ -368,8 +368,6 @@ void profilerStep() {
 #ifdef USE_KINETO
   if (libkineto::api().isProfilerInitialized()) {
     libkineto::api().activityProfiler().step();
-  } else {
-    LOG(WARNING) << "Profiler is not initialized: skipping step() invocation";
```

Comment:
do we want LOG(INFO) or completely removing the log?

Reply:
oh yea i can do that!